client/my-sites/comments/comment/comment-author.jsx

```diff
@@ -34,10 +34,7 @@ export class CommentAuthor extends Component {
 		isLinkTooltipVisible: false,
 	};
 
-	shouldComponentUpdate = ( nextProps, nextState ) =>
```

Comment:
Good call. This attempted optimisation seems like overkill for what's mostly a leaf component.

Reply:
Yes! It also used the deep `isEqual` comparison, which is a red flag.